src/utils/index.js

```diff
@@ -0,0 +1,39 @@
+// This will take an object and for each (k,v) will return
+// v rounded such that the sum of all v is 100.
+export const roundObjectPercentages = dataMap => {
+  // This algorithm workson integers and we want 2 decimal places.
+  // So round up first.
+  let asArray = Object.entries(dataMap).map(keyValue => {
+    keyValue[1] = keyValue[1] * 100
+    return keyValue
+  })
+
+  // Calculate difference from 100%
+  var margin =
+    10000 -
```

Comment:
why 10000?

Reply:
I'll make this clearer. Scaling for 100%, accounting for 2 extra decimal places.